ts/components/governance/treasury_summary.tsx

```diff
@@ -0,0 +1,22 @@
+import marked, { Token, Tokens } from 'marked';
+import React from 'react';
+
+import { Heading, Paragraph } from 'ts/components/text';
+
+export const TreasurySummary: React.FC<{description: string}> = ({ description }) => {
+  const tokens = marked.lexer(description);
+  const heading = tokens.find((token: Token) => (token as Tokens.Heading).type === 'heading' && (token as Tokens.Heading).depth === 1);
+  const paragraph = tokens.find((token: Token) => (token as Tokens.Paragraph).type === 'paragraph');
+  let summary = '';
+  // @ts-ignore
```

Comment:
just curious, how this is working + why the ignore is needed.  using the find above, isn't paragraph a singular token of type Tokens.Paragraph? (looking at the types, I would think only Tokens.Link has tokens property).

Reply:
Yeah, the types from library are misleading. The paragraph does contain more tokens broken down into text and links each of which have a `.text` property on them that contains the raw text.